src/config/dynamic-theme-fixes.config

```diff
@@ -17070,6 +17070,18 @@ INVERT
 
 ================================
 
+zombo.com
+
+CSS
+body {
+    background: none !important;
```

Comment:
Why should the background be removed?

Reply:
I think it fits better when there's no gradient

Reply:
So, the color of the gradient is fine and modified correctly, the original site has it as well, so it shouldn't be removed.